src/chttpd_auth_cache.erl

```diff
@@ -91,6 +102,12 @@ handle_info({'DOWN', _, _, Pid, Reason}, #state{changes_pid=Pid} = State) ->
     {noreply, State#state{last_seq=Seq}};
 handle_info({start_listener, Seq}, State) ->
     {noreply, State#state{changes_pid = spawn_changes(Seq)}};
+handle_info({gen_event_EXIT, {config_listener, ?MODULE}, _Reason}, State) ->
+    erlang:send_after(5000, self(), restart_config_listener),
```

Comment:
I used the same value as for restarting the config listener over in https://github.com/apache/couchdb-couch/blob/master/src/couch_auth_cache.erl#L235 - I don't know exactly why 5000 was used there, I'd guess possibly to limit the frequency in the event of repeated gen_event_EXIT messages?